client/actions/editActions2.js

```diff
@@ -284,6 +287,65 @@ export const saveArticlePending = () => {
   }
 }
 
+export const onAddFeature = (model, item) => {
```

Comment:
Funny I had a conversation with Jun yesterday about our use of the word "Feature" on publishing and here is another instance! Could we be a little more verbose here? `onAddFeaturedEntity` or `onAddLinkedEntity`? And maybe below to match Mentioned too?

Reply:
Yes great suggestion